examples/variable/app.css

```diff
@@ -0,0 +1,88 @@
+
+/* mini reset */
+* { margin: 0; padding: 0; }
+
+body,
+html {
+  height: 100%;
+  overflow: hidden;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+}
+
+header,
+footer {
+  height: 50px;
+  background: #eee;
+}
+
+header a {
+  display: block;
+  float: right;
+  width: 30px;
+  margin: 0 20px;
+
+  font-size: 2em;
+  line-height: 50px;
+  text-decoration: none;
+}
+
+#content {
+  position: relative;
+  flex: 1;
+}
+
+ul {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+
+  background: linear-gradient(to top, #bcbcbc, #bcbcbc 1px, white 1px, white);
+  background-position: 0 -88px;
+  background-size: 100% 88px;
+  background-repeat: repeat-y;
+
+  overflow: hidden;
+}
+
+ul li {
+  width: 100%;
+  height: 88px;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  -moz-user-select: none;
+
+  background: white;
+  border-bottom: 1px #bcbcbc solid;
+
+  z-index: 10;
+}
+
+ul li h3 {
+  margin: 0;
+  padding: 9px 6px 3px 6px;
+  background: white;
+
+  color: #424242;
+}
+
+ul li p {
+  margin: 0;
+  padding: 3px 6px 10px 6px;
+  background: white;
+
+  color: #797979;
+}
+
+ul li[data-populated=false] {
+  opacity: 0;
+}
```

Comment:
`display: none` may be a better choice as Gecko won't need to lay them out and we don't risk them getting in the way. This will invalidate layout, but we'rve likely already scheduled reflow after `reloadData()`.